docs/pages/blog/2021-q1-update.md

```diff
@@ -0,0 +1,220 @@
+- 🚨 We have started to work on breaking changes to reach a stable stage.
+- 💾 We have extended the support of @material-ui/core to handle v4 and v5 at the same time.
+  In the future, we might desynchronize the release version of _mui/material-ui_ and _mui/material-ui-x_.
+  For Material-UI X, we will likely need to release breaking changes at a higher frequency: every six months.
+- 🎛 We have added a column selector.
+
+  <img src="/static/blog/2021-q1-update/column-selector.png" alt="" style="width: 518px; margin-bottom: 16px;" />
+
+### Design kits
+
+#### Figma
+
+We have migrated all the components to leverage the [Figma variant](https://help.figma.com/hc/en-us/articles/360056440594-Create-and-use-variants) feature.
+We have also taken this as an opportunity to polish the components:
+
+- Improve usage of auto-layout
+- Fix theme design token usage
+- Add the Autocomplete component
+
+#### Adobe XD
+
+We have made Adobe XD available with a first release.
+
+#### Sketch
+
+We have redesigned most of the components from scratch to make sure all components show their CSS properties using inspection tools such as [Sketch Cloud Inspect tool](https://www.sketch.com/blog/2020/01/29/introducing-cloud-inspector-free-developer-handoff-in-the-browser/).
+
+## Company
+
+### Handbook
+
+The most important change inside the organization is the introduction of a company Handbook.
+The company is distributed and operates among different timezones.
+Per the nature of our operation, and at the pace we are growing, we needed an effective and efficient way to share the organizing processes and culture.
+Per our transparency value, all the sections of the Handbook that don't contain sensitive information are [publicly available](https://www.notion.so/Handbook-f086d47e10794d5e839aef9dc67f324b).
+
+This Handbook is the single **source of truth**. It's meant to be updated by anyone, to say in sync with how we do things.
+
+### Growth between Q4 2020 and Q1 2021
+
+- 📦 From 6.4M to 8.6M downloads per month on npm.<br />
+  It seems that React keeps taking market shares inside the frontend ecosystem.
+  For instance, it's growing faster than Vue or Angular.
+  All the indicators points to it: [StackOverflow questions](https://insights.stackoverflow.com/trends?tags=vue.js%2Creactjs%2Cangular), [stars](https://bestofjs.org/projects?sort=monthly), [downloads](https://npm-stat.com/charts.html?package=react-dom,@angular/core,vue), [Traffic on the documentation](https://www.similarweb.com/website/reactjs.org).
+  Our strategy to focus on React only seems to pay off.
+
+- ⭐️ From 63.8k to 67.2k stars, leave us yours [🌟](https://github.com/mui-org/material-ui).
+  We have seen a significant influx of more stars than usual this quarter.
+- 👨‍👩‍👧‍👦 From 2,052 to 2,141 contributors on GitHub. We add on average one new contributor every day.
+- 💰 Grew gross monthly revenue by 68% (+4% weekly growth rate). We have never grown this fast.
```

Comment:
```suggestion
```

Reply:
Afraid of raising interest from the competition? You would prefer showing a lagging metric which is the headcount? My hope is that this figure helps to hire.